aspnet/mvc/controllers/routing.rst

```diff
@@ -1,8 +1,140 @@
-.. include:: /../common/stub-topic.txt
+Routing to Controller Actions
+=============================
 
-|stub-icon| Routing to Controller Actions
-=========================================
+By `Rachel Appel`_
 
-.. include:: /../common/stub-notice.txt
+In this article:
 
-.. _issue: https://github.com/aspnet/Docs/issues/117
+.. contents:: Sections
```

Comment:
We should make sure we have the appropriate links here to the existing routing article which defines the non-MVC concepts already. We won't have a detailed description here about route constraints or the template syntax.